app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -187,6 +190,49 @@ class AdministratorControlsActivityTest {
     }
   }
 
+  @Test
+  fun testAdministratorControlsFragment_clickLogoutButton_displaysLogoutDialog() {
+    ActivityScenario.launch<AdministratorControlsActivity>(createAdministratorControlsActivityIntent(0)).use {
+      onView(withId(R.id.administrator_controls_list)).perform(scrollToPosition<RecyclerView.ViewHolder>(4))
+      onView(withId(R.id.log_out_text_view)).perform(click())
+      onView(withText(R.string.log_out_dialog_message)).inRoot(isDialog()).check(matches(isDisplayed()))
+      onView(withText(R.string.log_out_dialog_okay_button)).inRoot(isDialog()).check(matches(isDisplayed()))
+      onView(withText(R.string.log_out_dialog_cancel_button)).inRoot(isDialog()).check(matches(isDisplayed()))
+    }
+  }
+
+  // TODO(#762): Replace ProfileActivity to LoginActivity once it is added.
```

Comment:
Nit: `// TODO(#762): Replace [ProfileChooserActivity] to [LoginActivity] once it is added.`

Reply:
Done. I have kept name as `ProfileChooserActivity `instead of `ProfileChooserFragment` because I am creating a new issue of renaming every file related to `ProfileActivity `to `ProfileChooserActivity `so it makes sense.

Reply:
Makes sense. Thanks.